config/jenkins-release.yaml

```diff
@@ -479,10 +479,12 @@ controller:
           authorizationStrategy:
             globalMatrix:
               permissions:
-                - "Overall/Administer:release-core"
-                - "Overall/SystemRead:authenticated"
-                - "Overall/Read:authenticated"
-                - "Job/Read:authenticated"
+                - "GROUP:Overall/Administer:release-core"
+                - "GROUP:Overall/Administer:admins"
+                - "GROUP:Overall/Administer:jenkins-admins"
```

Comment:
@daniel-beck @timja again in common with @dduportal we were wondering if the "jenkins-admin" group should be added to release.ci.jenkins.io.
WDYT?